[2024-07-19T19:39:23.130+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-19T19:39:23.159+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_fill_f_101_round_f.fill_dm_account_balance_f manual__2024-07-19T19:38:54.682703+00:00 [queued]>
[2024-07-19T19:39:23.171+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_fill_f_101_round_f.fill_dm_account_balance_f manual__2024-07-19T19:38:54.682703+00:00 [queued]>
[2024-07-19T19:39:23.172+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-19T19:39:23.191+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fill_dm_account_balance_f> on 2024-07-19 19:38:54.682703+00:00
[2024-07-19T19:39:23.199+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'run_fill_f_101_round_f', 'fill_dm_account_balance_f', 'manual__2024-07-19T19:38:54.682703+00:00', '--job-id', '627', '--raw', '--subdir', 'DAGS_FOLDER/—Åreate_and_fill_f101_round_f.py', '--cfg-path', '/tmp/tmpai7hhljv']
[2024-07-19T19:39:23.202+0000] {standard_task_runner.py:91} INFO - Job 627: Subtask fill_dm_account_balance_f
[2024-07-19T19:39:23.225+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=17953) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-19T19:39:23.226+0000] {standard_task_runner.py:63} INFO - Started process 17956 to run task
[2024-07-19T19:39:23.242+0000] {task_command.py:426} INFO - Running <TaskInstance: run_fill_f_101_round_f.fill_dm_account_balance_f manual__2024-07-19T19:38:54.682703+00:00 [running]> on host 7c7e02e57379
[2024-07-19T19:39:23.316+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_fill_f_101_round_f' AIRFLOW_CTX_TASK_ID='fill_dm_account_balance_f' AIRFLOW_CTX_EXECUTION_DATE='2024-07-19T19:38:54.682703+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-19T19:38:54.682703+00:00'
[2024-07-19T19:39:23.317+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-19T19:39:23.335+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-07-19T19:39:23.344+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-07-19T19:39:23.355+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-07-19T19:39:23.391+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-19T19:39:23.392+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-19T19:39:23.401+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=run_fill_f_101_round_f, task_id=fill_dm_account_balance_f, run_id=manual__2024-07-19T19:38:54.682703+00:00, execution_date=20240719T193854, start_date=20240719T193923, end_date=20240719T193923
[2024-07-19T19:39:23.441+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-19T19:39:23.461+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-19T19:39:23.464+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
